src/main/java/com/nfl/glitr/registry/TypeRegistry.java

```diff
@@ -240,6 +216,50 @@ public GraphQLObjectType createRelayMutationType(Class clazz) {
         return builder.build();
     }
 
+    /**
+     * check if the given class contains property marked with @{@link GlitrQueryComplexity} annotation, if found
+     * add it to query complexity multipliers map.
+     * <p> processing of incoming {@code clazz} is based on recursion and moving into deep of property types and finished
+     * if the current processing property is the one of the primitives or the type is an {@code Object}
+     * or the type of property is one of the instance of {@code Map}
+     *
```

Comment:
... is based on recursion and **moving into deep of property types and finished if the current processing property is the one of the primitives** or the type is...

bolded part of the comment that is unclear, please revise.